catalog/rocks/validate.go

```diff
@@ -4,13 +4,16 @@ import (
 	"errors"
 	"fmt"
 	"regexp"
+	"strings"
 
 	"github.com/treeverse/lakefs/graveler"
 	"github.com/treeverse/lakefs/ident"
 )
 
 const (
 	MaxPathLength = 1024
+
+	minControlCharCode = 40
```

Comment:
This is the char code for '8'.  The closest I can think of is `maxControlCharCode = 040-1`, assuming golang has octals.

Reply:
True - it should be 040 or octa40